frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -85,7 +125,8 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
           <strong>{{ namespace }}</strong>. Removing the Operator will not remove any of its custom
           resource definitions or managed resources. If your Operator has deployed applications on
           the cluster or configured off-cluster resources, these will continue to run and need to be
-          cleaned up manually.
+          cleaned up manually. Any console plugins provided by this operator will be disabled and
+          removed.
```

Comment:
We shouldn't show this in the message to users who don't actually have authority to patch the config and remove the plugin.

cc @itsptk for design input

Reply:
@itsptk will be on paternity leave starting tomorrow. This was his last message to me about this. 

> I mentioned at our weekly stakeholder call with Sam and Ali that we were trying to sort out this messaging. Sam mentioned he thought it was possible but wasn't sure how common it would be, but did want to follow up at an architects meeting to discuss this scenario more in general. He said he would follow back up with us.